examples/classifier/CL4_apply_DTW_based_kmeans_clustering.py

```diff
@@ -0,0 +1,87 @@
+# Standard library imports
+import sys
+new_path = sys.path[0][:-len("/examples/timeseries")]
+sys.path.append(new_path)
+
+# Third party imports
+import itertools
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from sklearn import cluster
+
+# Local application imports
+from scifin import timeseries as ts
+from scifin import montecarlo as mc
+from scifin import classifier as cl
+
+# Generate two kinds of processes
+N = 50
+mc_MA = mc.generate_series(n=N, names_base="MA-", series_model=ts.moving_average,
+                         start_date="2000-01-01", end_date="2020-01-01", frequency='M',
+                         cst=1., order=2, coeffs=[0.5, 0.2], sigma=0.4)
+
+mc_Garch = mc.generate_series(n=N, names_base="Garch-", series_model=ts.garch,
+                         start_date="2000-01-01", end_date="2020-01-01", frequency='M',
+                         cst=0.2, order_a=1, coeffs_a=[0.2], order_sig=1, coeffs_sig=[0.1])
+
+# Combining them together
+mc1 = []
+for tsi in mc_MA:
+    mc1.append(tsi)
+for tsi in mc_Garch:
+    mc1.append(tsi)
```

Comment:
There are simpler ways to append lists together, e.g.

1)  use * to turn the list to a generator
``
mc1 = [*mc_MA, *mc_Garch]  
``
2) use .extend() method
```
mc1 = mc_MA.copy()
mc1.extend(mc_Garch)
```

Reply:
Oh that's really nice this way! I'm going to use your first suggestion, very elegant!

Reply:
Just for the fun of it, there is a third way:
```
mc_MA + mc_Garch
```

Reply:
Thx.